recipes/r-recetox-aplcms/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$R CMD INSTALL --build .
```

Comment:
you can put this also in the meta.yaml file as well?

Reply:
Sure, I've just moved it to the meta.yaml